Python/pywarpx/callbacks.py

```diff
@@ -93,8 +93,11 @@ def __call__(self,*args,**kw):
 
     def clearlist(self):
         self.funcs = []
+        libwarpx.libwarpx_so.warpx_clear_callback_py(
+            ctypes.c_char_p(self.name.encode('utf-8'))
+        )
 
-    def __nonzero__(self):
+    def __bool__(self):
         "Returns True if functions are installed, otherwise False"
```

Comment:
While I see this, I think function docs need to be written like this in Python:
```suggestion
        """Returns True if functions are installed, otherwise False"""
```